_config.yml

```diff
@@ -6,15 +6,15 @@
 # 'jekyll serve'. If you change this file, please restart the server process.
 
 # Site settings
-title: Octave
+title: GNU Octave
 email: maintainers@octave.org
 description: > # this means to ignore newlines until "baseurl:"
-  Octave is a programming language for scientific computing.
-baseurl: "/octave-web" # the subpath of your site, e.g. /blog
-url: "//www.octave.org" # the base hostname & protocol for your site
-wiki_url: "//wiki.octave.org"
-faq_url: "//wiki.octave.org/faq"
-docs_url: "//www.gnu.org/software/octave/doc/interpreter/"
+  GNU Octave is a programming language for scientific computing.
+baseurl: "/software/octave/new" # the subpath of your site, e.g. /blog
```

Comment:
Load the value from the environment, e.g. the shell where it was launched,
falling back to a default if nothing is found.

On Thu, Jul 14, 2016 at 12:55 AM Kai Torben Ohlhus <notifications@github.com>
wrote:

> In _config.yml
> <https://github.com/alexkrolick/octave-web/pull/2#discussion_r70763630>:
>
> >  email: maintainers@octave.org
> >  description: > # this means to ignore newlines until "baseurl:"
> > -  Octave is a programming language for scientific computing.
> > -baseurl: "/octave-web" # the subpath of your site, e.g. /blog
> > -url: "//www.octave.org" # the base hostname & protocol for your site
> > -wiki_url: "//wiki.octave.org"
> > -faq_url: "//wiki.octave.org/faq"
> > -docs_url: "//www.gnu.org/software/octave/doc/interpreter/"
> > +  GNU Octave is a programming language for scientific computing.
> > +baseurl: "/software/octave/new" # the subpath of your site, e.g. /blog
>
> What do you mean by environment variable?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/alexkrolick/octave-web/pull/2/files/78a831e8e93c94ade879f87afcef98b8687d1867#r70763630>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABf7U9jhw4JEpNMOkKzss2kfEnykU2sIks5qVet6gaJpZM4JK2_L>
> .
>